linode/user/tmpl/grants_update.gotf

```diff
@@ -1,14 +1,12 @@
 {{ define "user_grants_update" }}
 
-{{ template "e2e_test_firewall" . }}
-
 # TODO: reuse no image instance template defined in instance
 resource "linode_instance" "foobar" {
     label = "{{.InstLabel}}"
     group = "tf_test"
     type = "g6-nanode-1"
     region = "us-east"
-    firewall_id = linode_firewall.e2e_test_firewall.id
```

Comment:
So this instance won't be covered by the firewall?

Reply:
This instance won't be booted up since the test does not require a booted instance. E.g. `booted=false`

Reply:
Fyi, firewall is not required for instances that are not booted for 48 hrs